BCVaccineValidator/Classes/Services/ImmunizationService.swift

```diff
@@ -41,6 +41,9 @@ class ImmunizationService {
     }
     
     private func getImmunizationStatus(for payload: DecodedQRPayload, using rulesSet: RuleSet) -> ImmunizationStatus? {
+        if payload.isExempt() {
```

Comment:
'guard' instead of 'if' would be better.

Reply:
@AfsarFresh  no we need to proceed if condition fails

Reply:
I get the point. But as a best practice, its better to use the 'guard' to return from a function in the middle.
We can just have !payload.isExempt() for the 'guard'.

Reply:
yes, it's the best practice in general, but guard is used to "guard" (protect) the following logic not to fulfill specific condition, it's not the case here because you early got the status (return type)
but yes, revising the condition works well.